### PR TITLE
added java api binding for ccb learner

### PR DESCRIPTION
**Pull request by zhxiaogg**
I managed to add support for ccb input/output format support in java wrapper, without knowing much about the vw internal implementation. 

I need your helps on reviewing this to verify those changes are as I expected, thanks!

### CODE REVIEW

**Comment by zhxiaogg** [comment]
hi @jackgerrits, I don't know who should I ask for, but could you help reviewing this or @ somebody else?

**Review by olgavrou — APPROVED** [review]
Hi @zhxiaogg thank you for your contribution. This change looks reasonable to me, however I haven't worked on this part of the codebase so it might be best to wait for another review too

**Review by jackgerrits — APPROVED** [review]
Looks good to me. Thanks for implementing this!

**Pull request merged**
